google/cloud/aiplatform/datasets/tabular_dataset.py

```diff
@@ -33,6 +39,151 @@ class TabularDataset(datasets._Dataset):
         schema.dataset.metadata.tabular,
     )
 
+    @property
+    def column_names(self) -> List[str]:
```

Comment:
I get this log:
```INFO:google.resumable_media._helpers:No MD5 checksum was returned from the service while downloading```

source; https://github.com/googleapis/google-resumable-media-python/blob/master/google/resumable_media/_helpers.py#L39

when using this property. Can we suppress or resolve this log?

Reply:
Will look into it.

Reply:
I don't get this log, do you have repro steps?

Reply:
```Python
for ds in aiplatform.TabularDataset.list():
  print(ds.column_names)
```

Might be specific to the version of the library. I see that log with:
```
google-cloud-storage==1.35.1
google-resumable-media==1.2.0
```

What versions are you testing with?

Reply:
I could reproduce it with on GCS datasets.

I added code to suppress logging during the GCS download.